CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Features:
 - [#1340](https://github.com/rails-api/active_model_serializers/pull/1340) Add support for resource-level meta. (@beauby)
 
 Fixes:
+- [#1622](https://github.com/rails-api/active_model_serializers/pull/1622) Fix fragment caching inherited serializers to use distinct per-serializer caches. (@lserman)
```

Comment:
Yeah, but that's not the only thing it solve actually. If you define two serializers with the same model, you would get the same issue. That's actually what prevented me to properly refactor fragment cache in #1565.